src/HornedBeast.css

```diff
@@ -0,0 +1,15 @@
+article {
+margin-top: 1vw;
+margin-left: 2vw;
+width: 21%;
+display: inline-block;
+}
+
```

Comment:
Great job on the lab 👍 

- Grab a TA when you can to check your styles. The app is on its way to be responsive, but we need to tweak some styles. You may need to look at your `article `width vs the elements that are sitting inside..the article width is 21% and the images are overlapping one another.
-As the screensize shrinks, the images will show in column and in full width should be in a row with no overlap 👍